R/coord-sf.R

```diff
@@ -235,6 +235,17 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
     # override graticule labels provided by sf::st_graticule() if necessary
     graticule <- self$fixup_graticule_labels(graticule, scale_x, scale_y, params)
 
+    # Convert graticule to viewscales for axis guides
+    viewscales <- Map(
+      view_scales_from_graticule,
+      scale = list(x = scale_x, y = scale_y, x.sec = scale_x, y.sec = scale_y),
+      aesthetic = c("x", "y", "x.sec", "y.sec"),
+      label = self$label_axes[c("bottom", "left", "top", "right")],
+      MoreArgs = list(graticule = graticule, bbox = bbox,
+                      label_graticule = self$label_graticule)
```

Comment:
Just FYI: I think the formatting here is not tidyverse style.

Reply:
Good catch.